imap_processing/cdf/config/imap_idex_l2a_variable_attrs.yaml

```diff
@@ -28,19 +25,17 @@ impact_charge_base: &impact_charge_base
   DISPLAY_TYPE: time_series
   FORMAT: F20.4
   FILLVAL: *double_fillval
-  LABLAXIS: "Impact Charge"
   DICT_KEY: SPASE>Particle>ParticleType:Dust>ParticleQuantity:Current>Qualifier:Derived # TODO quantity should be charge but there is no SPASE charge
 
 mass_estimate_base: &mass_estimate_base
-  UNITS: kg
+  UNITS: u
```

Comment:
Is "u" a valid unit, or does it need to be "amu" (presuming it is atomic mass unit)